components/features/galleryAssets/web/sell-action.tsx

```
@@ -0,0 +1,196 @@
+
+	const handleSellComplete = async () => {
+		if (!profile) return;
+		try {
+			const walletProvider = getProviderStrategy(profile.providerName);
+			const signatureHex = await walletProvider.signTx(
+				sellStartData?.sighash_hex_to_sign ?? "",
+				{
+					address: profile?.tapasAddress ?? "",
+				},
+			);
+			debugger;
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)